Sources/Controllers/Settings/GeneralSettings/OAProfileGeneralSettingsParametersViewController.m

```diff
@@ -395,6 +447,25 @@ - (UITableViewCell *)getRow:(NSIndexPath *)indexPath
         }
         return cell;
     }
+    if ([cellType isEqualToString:[OASimpleTableViewCell getCellIdentifier]])
```

Comment:
If you stopped using OASettingsTitleTableViewCell you should remove the code above